admin/class-boldgrid-backup-admin-core.php

```diff
@@ -741,6 +741,8 @@ public function __construct() {
 
 		$this->remote = new Boldgrid_Backup_Admin_Remote( $this );
 
+		$this->folder_exclusion = new Boldgrid_Backup_Admin_Folder_Exclusion( $this );
```

Comment:
Why was this moved? If for no valid reason, can we undo this change. No reason to have it associated with fixing this issue if it didn't.

Reply:
It had to be moved because the 'folder_exclusion' property was undefined in the 'jobs' class when it was later down the list.